lib/core-common/src/types.ts

```diff
@@ -235,12 +236,21 @@ export interface StorybookConfig {
   >;
   core?: CoreConfig;
   logLevel?: string;
-  /**
-   * Allows to disable deprecated implicit PostCSS loader.
-   */
   features?: {
+    /**
+     * Allows to disable deprecated implicit PostCSS loader.
+     */
     postcss?: boolean;
+
+    /**
+     * Build stories.json automatically on start/build
+     */
     buildStoriesJson?: boolean;
+
+    /**
+     * Activate preview of CSF v3.0
+     */
```

Comment:
Does it make sense to add a tsdoc [experimental](https://tsdoc.org/pages/tags/experimental/) flag?
On the same note, would be nice if `Meta` had its type updated to have the `render` property, also under experimental flag